docs/intro/overview.rst

```diff
@@ -0,0 +1,18 @@
+===============
+Getting started
+===============
+
+The goal of this project is to make reusable Page Objects that separates
+extraction logic from crawling. They could be easily tested and distributed
+across different projects. Also, they could make use of different backends,
+for example, acquiring data from Splash and Auto Extract API.
```

Comment:
Could you please add links for both Splash and AutoExtract?

Reply:
Done.